flow.js

```diff
@@ -77,32 +77,34 @@ class Flow {
       'repair-description-leak-source': {prevStep: 'repair-description-leak-inside', nextStep: 'repair-description'},
       'repair-leak-description-electrics-emergency': {prevStep: 'repair-description-leak-electrics'},
-      'repair-description': {prevStep:'' , nextStep: 'contact-person'},//need to investigate this as there are numerous prev steps, but it might just work
+      'repair-description': {prevStep:'repair-kitchen-cupboard-problems', nextStep: 'contact-person'},//need to investigate this as there are numerous prev steps, but it might just work
       'contact-person': {prevStep: 'repair-description', nextStep:'contact-details'},
       'contact-details': {prevStep: 'contact-person', nextStep: 'repair-availability'},
-      'repair-availability': {prevStep: 'personal-details', nextStep: 'summary'},
+      'repair-availability': {prevStep: 'contact-details', nextStep: 'summary'},
+      'summary': {prevStep: 'repair-availability', nextStep:''},//need to investigate this as there are numerous prev steps, but it might just work
+
       // 'contact-details-appointment': { prevStep: 'repair-availability', nextStep: 'appointment-playback'},
       // 'appointment-playback': {prevStep: 'contact-details-appointment', nextStep: [
       //   {condition: 'change-appointment', nextStep:'repair-availability'},
       //   {condition: 'keep-appointment', nextStep: 'contact-details'}
       // ]},
-      'summary': {prevStep: 'appointment-playback', nextStep: [
-        {condition: 'change-repair-address', nextStep: 'postcode'},
-        {condition: 'change-contact-details', nextStep: 'contact-details'},
-        {condition: 'change-phone-number', nextStep: 'contact-details-appointment'},
-        {condition: 'change-location', nextStep: 'repair-location'},
-        {condition: 'change-issue', nextStep: 'repair-kitchen-types'},
-        {condition: 'change-description', nextStep: 'repair-description'},
-        {condition: 'change-appointment-date', nextStep: 'repair-availability'},
-        {condition: 'change-appointment-information', nextStep: 'personal-details'},
-        {condition: 'confirm-and-report', nextStep: 'confirmation'}
-      ]},
+      // 'summary': {prevStep: 'appointment-playback', nextStep: [
+      //   {condition: 'change-repair-address', nextStep: 'postcode'},
+      //   {condition: 'change-contact-details', nextStep: 'contact-details'},
+      //   {condition: 'change-phone-number', nextStep: 'contact-details-appointment'},
+      //   {condition: 'change-location', nextStep: 'repair-location'},
+      //   {condition: 'change-issue', nextStep: 'repair-kitchen-types'},
+      //   {condition: 'change-description', nextStep: 'repair-description'},
+      //   {condition: 'change-appointment-date', nextStep: 'repair-availability'},
+      //   {condition: 'change-appointment-information', nextStep: 'personal-details'},
+      //   {condition: 'confirm-and-report', nextStep: 'confirmation'}
+      // ]},
```

Comment:
do we need these comments here?

Reply:
this is because there is already a summary page before this. I commented because it was conflicting and wanted to defer the renaming for whoever pick this up